### DISCUSSION TITLE
從租片公司，到全球追劇領頭羊，擁有2億付費用户，Netlflix到底贏在哪？

### DISCUSSION THREAD

**Discussion by 107602505** [1101-GS4719-C/Python-Community-GS4719-C#285] [General]
大家應該都會NETFLIX不陌生吧，甚至也常常花很多時間在這個平台上，看完這個影片真的對這個以內容為王的平台又更加佩服了
https://www.youtube.com/watch?v=mjlNa0BV-OM

**Comment by JAIMEHUNG** [comment]
影片很淺顯易懂!NETFLIX真的把顧客綁得牢牢的，因為它有確實抓到顧客的需求。

**Comment by zixunn** [comment]
由次可以得出，想賺錢不只要有頭腦，還要有時勢提供的機會，看完影片不禁讓人佩服老哈眼光的獨辣，NEFIX走在了個性化推薦技術的前端，後來又走在了自製節目的前端，因為老哈不斷的創新，才讓NEFIX能夠不被時間淘汰。

**Comment by syt-110** [comment]
可能是符合客戶喜好、可以步用出門租或者還光碟?

**Comment by HSU-YU-JEN** [comment]
之前有看到類似報導，nexlflix之所以能成功，就是打破平台方出品創作的慣例，轉而以與導演或製片人合作的方式，產出多元且具地區性的內容

**Comment by MAKERRRofKEVIN** [comment]
Netflix的企業精神一直以來都令我相當嚮往，有興趣的同學也可以看看「零規則」裡面談到了很多Netflix成功的關鍵，想進一步認識Netflix的話千萬不能錯過這本佳作

**Comment by Isongzhe** [comment]
之前百視達有機會把Netflix買下來，可惜沒跟上這樣串流平台的潮流。

**Comment by Joan0518** [comment]
這個影片講解得很清楚!
有這樣的商業頭腦加上衝勁，造就了Netflix的成功
不禁令人好奇最近起來的Disney+抓住顧客的方法是如何
